### ISSUE TITLE
advanced options not appearing

### ISSUE THREAD

**Issue by JackTheCr33per** [TheCBProject/NotEnoughItems#118]
in my world, the options for NEI like healing yourself, changing the time of day, and changing gamemode, they do not appear in my inventory, but all the items do. i have tried every single option on NEI, but they still do not appear.
![nei](https://user-images.githubusercontent.com/30801905/29030336-a5fddee8-7b58-11e7-9452-a1286599a144.png)

**Comment by DatrixTHLK** [comment]
I've tried the options . Magnet mode does not work at this point in time. Here are the steps to follow:

1. Go to your inventory
2. Go down to where you see options and click on that
3. Go to Inventory button click on that
4. Click on Global then the letter W should show up on each of the options
5. Click on the w to enable them
6. Click on the global option at top right again and the options should show up.

If I missed some steps I will have to redo what I did before and see if I can figure out what I missed.
